squarelet/organizations/viewsets.py

```diff
@@ -15,8 +15,7 @@
 
 
 class OrganizationViewSet(viewsets.ModelViewSet):
-    # remove _plan after clients are updated
-    queryset = Organization.objects.select_related("_plan")
+    queryset = Organization.objects.prefetch_related("subtypes")
```

Comment:
`_plan` is no longer needed, but prefetching `subtypes` prevents an additional SQL query for every result returned in the list view